packages/nc-gui-v2/composables/useViewFilters.ts

```diff
@@ -129,17 +125,11 @@ export function useViewFilters(
   }
 
   const saveOrUpdate = async (filter: Filter, i: number, force = false) => {
-    if (isPublic.value) {
-      filters.value[i] = { ...filter }
-
-      filters.value = [...filters.value]
-
-      return
-    }
     if (!view?.value) return
 
-    if (!isUIAllowed('filterSync')) {
```

Comment:
users without `filterSync` shouldn't be able to update the filters in meta.

Reply:
@wingkwong `nestedMode` includes that